pkg/bbgo/activeorderbook.go

```diff
@@ -92,60 +92,50 @@ func (b *ActiveOrderBook) waitAllClear(ctx context.Context, waitTime, timeout ti
 	}
 }
 
-// Cancel cancels the given order from activeOrderBook gracefully
-func (b *ActiveOrderBook) Cancel(ctx context.Context, ex types.Exchange, order types.Order) error {
-	if !b.Exists(order) {
-		return fmt.Errorf("cannot find %v in orderbook", order)
+func (b *ActiveOrderBook) CancelNoWait(ctx context.Context, ex types.Exchange, orders ...types.Order) error {
```

Comment:
can you convert the NoWait  to an option defined in ActiveOrderBook? like activeOrders.SetNoWait(true)

Reply:
we might need both operations on the same activeOrderbook in a single strategy.  
for example, in emergency exit, we could like all the orders to be successfully cancelled, so we call GracefulCancel, while in high frequency trading, we still need CancelNoWait.